<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvnNewProjectOptionsDialog</class>
 <widget class="QDialog" name="SvnNewProjectOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Project from Repository</string>
  </property>
  <property name="whatsThis">
   <string>&lt;b&gt;New Project from Repository Dialog&lt;/b&gt;
&lt;p&gt;Enter the various repository infos into the entry fields. These values are used, when the new project is retrieved from the repository. If the checkbox is selected, the URL must end in the project name. A repository layout with project/tags, project/branches and project/trunk will be assumed. In this case, you may enter a tag or branch, which must look like tags/tagname or branches/branchname. If the checkbox is not selected, the URL must contain the complete path in the repository.&lt;/p&gt;
&lt;p&gt;For remote repositories the URL must contain the hostname.&lt;/p&gt;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>&amp;Protocol:</string>
     </property>
     <property name="buddy">
      <cstring>protocolCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="protocolCombo">
     <property name="toolTip">
      <string>Select the protocol to access the repository</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="vcsUrlLabel">
     <property name="text">
      <string>&amp;URL:</string>
     </property>
     <property name="buddy">
      <cstring>vcsUrlPicker</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="E5PathPicker" name="vcsUrlPicker" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Enter the url path of the module in the repository (without protocol part)</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;URL&lt;/b&gt;&lt;p&gt;Enter the URL to the module. For a repository with standard layout, this must not contain the trunk, tags or branches part.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="vcsTagLabel">
     <property name="text">
      <string>&amp;Tag:</string>
     </property>
     <property name="buddy">
      <cstring>vcsTagEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="vcsTagEdit">
     <property name="toolTip">
      <string>Enter the tag the new project should be generated from</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;Tag in VCS&lt;/b&gt;
&lt;p&gt;Enter the tag name the new project shall be generated from. Leave empty to retrieve the latest data from the repository.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="TextLabel4">
     <property name="text">
      <string>Project &amp;Directory:</string>
     </property>
     <property name="buddy">
      <cstring>vcsProjectDirPicker</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="E5PathPicker" name="vcsProjectDirPicker" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Enter the directory of the new project.</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;Project Directory&lt;/b&gt;
&lt;p&gt;Enter the directory of the new project. It will be retrieved from 
the repository and be placed in this directory.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="layoutCheckBox">
     <property name="toolTip">
      <string>Select to indicate, that the repository has a standard layout (projectdir/trunk, projectdir/tags, projectdir/branches)</string>
     </property>
     <property name="text">
      <string>Repository has standard &amp;layout</string>
     </property>
     <property name="shortcut">
      <string>Alt+L</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>protocolCombo</tabstop>
  <tabstop>vcsUrlPicker</tabstop>
  <tabstop>vcsTagEdit</tabstop>
  <tabstop>vcsProjectDirPicker</tabstop>
  <tabstop>layoutCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SvnNewProjectOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SvnNewProjectOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
